config/value.go

```diff
@@ -363,8 +396,7 @@ func convertValue(value interface{}, targetType reflect.Type) (interface{}, erro
 			return reflect.ValueOf(target).Elem().Interface(), err
 		}
 	}
-
-	return nil, fmt.Errorf("can't convert %v to %v", valueType, targetType)
+	return nil, nil
```

Comment:
Did you see the refactor? I have explained this above https://github.com/uber-go/fx/pull/219#discussion_r99660210